contrib/debian/rules

```diff
@@ -70,6 +70,11 @@ override_dh_install:
 	rm -f debian/fwupd/usr/lib/*/fwupd-plugins-*/libfu_plugin_test_ble.so
 	rm -f debian/fwupd/usr/lib/*/fwupd-plugins-*/libfu_plugin_invalid.so
 	rm -f debian/fwupd/etc/fwupd/remotes.d/fwupd-tests.conf
+	mv debian/fwupd/usr/share/man/man1 debian/fwupd/usr/share/man/man8
+	for file in debian/fwupd/usr/share/man/man8/*.1; do \
+		file8=$${file%%.1}.8; \
+		mv -v $$file $$file8; \
```

Comment:
is this something we should do upstream?

Reply:
I was wondering the same thing.  At least wanted to sync up the packaging for now, and can revert it if we change it upstream too.

Reply:
Hmm, not convinced e.g. `fwupdmgr` is designed to be run by end users, rather than sysadmins -- you don't even need root.

Reply:
I guess that's true, I'm not sure why this lintian warning came up.